src/resources/postcss/views/full/_event-status.pcss

```diff
@@ -0,0 +1,70 @@
+/* -----------------------------------------------------------------------------
+* Event Status.
+* ----------------------------------------------------------------------------- */
+
+.tribe-events-status__text {
+	color: var(--tec-color-text-primary);
+	font-weight: var(--tec-font-weight-regular);
+	line-height: var(--tec-line-height-0);
+}
+
+.tribe-events-status__text-bold {
+	font-weight: var(--tec-font-weight-bold);
+}
+
+.tribe-events-status__text-red {
+	color: var(--tec-color-icon-error);
+}
+
+.tribe-events-status__text-alert-icon {
+	align-items: center;
+	display: inline-flex;
+}
+
+.tribe-events-status__text-alert-icon:before {
+	background-image: url('../images/alert.svg');
+	background-repeat: no-repeat;
+	background-size: var(--tec-spacer-3);
+	content: '';
+	display: inline-block;
+	height: var(--tec-spacer-3);
+	margin-right: var(--tec-spacer-1);
+	min-width: var(--tec-spacer-3);
+}
+
+/* -----------------------------------------------------------------------------
+ * Event Status (Canceled, Postponed)
+ * ----------------------------------------------------------------------------- */
+
+.tribe-events-status__label {
+	color: var(--tec-color-icon-error);
+}
+
+/* -----------------------------------------------------------------------------
+* List.
+* ----------------------------------------------------------------------------- */
+
+.tribe-events-calendar-day .tribe-common-g-row,
+.tribe-common-g-row.tribe-events-calendar-list__event-row {
+	position: relative;
+}
```

Comment:
A few things:
1. Is there another selector to use other than `.tribe-common-g-row` in the first selector? This is a common selector, better to select a `.tribe-events-calendar-day__` selector.
2. Do you need the `.tribe-common-g-row` selector in the second selector?
3. This should belong in skeleton rather than full styles.

Reply:
I removed this. Not sure it is even used.